src/main/java/xyz/gvital/todo/service/TodoService.java

```diff
@@ -0,0 +1,42 @@
+package xyz.gvital.todo.service;
+
+import jakarta.transaction.Transactional;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import xyz.gvital.todo.dto.TodoCreateDTO;
+import xyz.gvital.todo.dto.TodoResponseDTO;
+import xyz.gvital.todo.dto.TodoUpdateDTO;
+import xyz.gvital.todo.exception.EntityNotFoundException;
+import xyz.gvital.todo.mapper.TodoMapper;
+import xyz.gvital.todo.model.Todo;
+import xyz.gvital.todo.repository.TodoRepository;
+
+@Service
+@AllArgsConstructor
```

Comment:
This annotation is more common to initialize dependencies
```suggestion
@RequiredArgsConstructor
```